### PR TITLE
Docs: Adding and updating miscellaneous keyboard shortcuts

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Updates old keyboard shortcuts scattered around the docs, to match the new specification (fixing #9423)
- Unifies the formatting of keys across the entire docs
- Adds **Related keyboard shortcuts** sections to plugins (fixing #9423)

Mind that, as an exception, these changes update two docs versions: `next` and `12.0` (current).

[skip changelog]

### CODE REVIEW

File: docs/12.0/guides/cell-features/disabled-cells.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Disabling a cell makes the cell read-only or non-editable. Both have similar out
 
 ## Read-only columns
 
-In many use cases, you will need to configure a certain column to be read-only. This column will be available for keyboard navigation and <kbd>CTRL+C</kbd>. Editing and pasting data will be disabled.
+In many use cases, you will need to configure a certain column to be read-only. This column will be available for keyboard navigation and <kbd>**Ctrl**</kbd> + <kbd>**C**</kbd>. Editing and pasting data will be disabled.
```
Comment (aninde) [inline docs/12.0/guides/cell-features/disabled-cells.md:21]: I think you forgot to add `coping`.

**Review by aninde — CHANGES_REQUESTED** [review]
The fixes look good, both on the desktop browsers (Chrome 101.0.4951.41) and phones. Checked on Safari on the iPhone SE 2020 / iOS 15.4.1

File: docs/12.0/guides/cell-features/disabled-cells.md (hunk 1 above)
Comment (kirszenbaum) [reply docs/12.0/guides/cell-features/disabled-cells.md]: @aninde I didn't edit the descriptions, but sure, it should be there: [5de8328](https://github.com/handsontable/handsontable/pull/9448/commits/5de83281ab61c8bcdeb0b1bf783219bfe04c2402)

**Review by aninde — APPROVED** [review]
Looks great 👍

**Pull request merged**